src/group_theory/order_of_element.lean

```diff
@@ -17,17 +17,14 @@ This file defines the order of an element of a finite group. For a finite group
 
 ## Main definitions
 
-* `order_of` defines the order of an element `a` of a group `G`.
+* `order_of` defines the order of an element `a` of a group `G`, by convention its value is `0` if
+  `a` has infinite order.
 * `is_cyclic` is a predicate on a group stating that the group is cyclic.
 
 ## Main statements
 
 `is_cyclic_of_prime_card` proves that a finite group of prime order is cyclic.
 
-## Implementation notes
-
-`order_of` is currently only defined for finite multiplicatively written groups.
```

Comment:
I think it would be good to keep a comment warning that this is only for multiplicative groups.